app/models/concerns/employed_income.rb

```diff
@@ -55,6 +55,14 @@ def employed_income_payment_types
     end
   end
 
```

Comment:
Moving to concerns allows us to share code between Structs and ActiveRecord objects.,